src/OpenTelemetry/.publicApi/Stable/PublicAPI.Unshipped.txt

```diff
@@ -25,4 +25,8 @@ OpenTelemetry.ReadOnlyFilteredTagCollection.Enumerator.Enumerator() -> void
 OpenTelemetry.ReadOnlyFilteredTagCollection.Enumerator.MoveNext() -> bool
 OpenTelemetry.ReadOnlyFilteredTagCollection.GetEnumerator() -> OpenTelemetry.ReadOnlyFilteredTagCollection.Enumerator
 OpenTelemetry.ReadOnlyFilteredTagCollection.ReadOnlyFilteredTagCollection() -> void
+*REMOVED*OpenTelemetry.Trace.SamplingResult.SamplingResult(OpenTelemetry.Trace.SamplingDecision decision, string! traceStateString) -> void
+*REMOVED*OpenTelemetry.Trace.SamplingResult.SamplingResult(OpenTelemetry.Trace.SamplingDecision decision, System.Collections.Generic.IEnumerable<System.Collections.Generic.KeyValuePair<string!, object!>>! attributes) -> void
```

Comment:
Are these still needed or can be removed?

Reply:
IMO needed. Changes should be propagated to released file while creating release PR.

Reply:
If all you are doing is changing nullable annotations, I think it is safe to remove these changes in the "Unshipped" file and just go update the lines in the shipped file. Change "!" to "?" for the things being made nullable. Nullable decorations don't really change the code just analysis of it there's nothing breaking with this change AFAIK.

Reply:
Fixed in d1c9071e70a70533cad69ad60c32cb43559d978e